saleor/graphql/page/filters.py

```diff
@@ -1,18 +1,29 @@
 import django_filters
+from django.db.models import Q
 from graphene_django.filter import GlobalIDMultipleChoiceFilter
 
 from ...page import models
 from ..core.filters import MetadataFilterBase
 from ..core.types import FilterInputObjectType
 from ..utils import resolve_global_ids_to_primary_keys
-from ..utils.filters import filter_by_query_param
-from .types import Page
+from .types import Page, PageType
 
 
 def filter_page_search(qs, _, value):
-    page_fields = ["content", "slug", "title"]
-    qs = filter_by_query_param(qs, value, page_fields)
-    return qs
+    if not value:
+        return qs
+    return qs.filter(
+        Q(title__trigram_similar=value)
+        | Q(slug__trigram_similar=value)
+        | Q(content__icontains=value)
+    )
+
+
+def filter_page_page_types(qs, _, value):
```

Comment:
Where is this function used?

Reply:
I didn't add the changes in `PageFilter` but it's fixed already.